static/sass/6-components/_footer.scss

```diff
@@ -1,6 +1,14 @@
 .c-site-footer {
-  .border--yellow_notch {
-    border-top: 2px solid $color-yellow-tribune;
-    margin: $size-xs 0;
+  
+  &__logo {
+    font-size: 4rem;
```

Comment:
Do you hate the idea of using an inline style to dictate the size of comically large icons like this one? I left a [comment](https://github.com/texastribune/ds-toolbox/pull/25#issuecomment-497519062) about that strategy in this PR -- not sure if it's a good one. Also doesn't matter much.

Reply:
I don't love it, but I understand what you mean. Tis changed.